### ISSUE TITLE
Unexpected HTTP response: 404

### ISSUE THREAD

**Issue by elbaulp** [fwilhe2/setup-kotlin#85]
Since yesterday I am getting a 404 error:

```
Run fwilhe2/setup-kotlin@main
  with:
    version: 1.4.21
/usr/bin/unzip -q /home/runner/work/_temp/ba4df20a-7c3e-493e-bff3-19111d9affd2
Error: Unexpected HTTP response: 404
```

I do not specify any version, so it is picking the default value.

Any ideas on what's going on? Why is receiving a 404?

**Comment by joffrey-bion** [comment]
Same problem here since last night.

Some notes:

* Fixing the version of this action to 0.0.6 (instead of `@main`) doesn't solve the issue
* Fixing the version of kotlin to `1.4.20` (instead of the default `1.4.21`) doesn't solve the issue

I wonder what changed recently.

**Comment by elbaulp** [comment]
Ah, yeah, forgot to mention, I also tried to change the version on the action itself and did not help either

**Comment by fwilhe2** [comment]
Thanks for reporting. JetBrains have changed the name of the kotlin/native archive which makes the download fail. Probably downloading native should be optional, but currently it is not.

You could use `0.0.5` of the action for now if you don't want to use native.

Otherwise, the url is fixed in https://github.com/fwilhe2/setup-kotlin/pull/86

This change is available in `main` and `0.0.7`.

**Comment by elbaulp** [comment]
Thanks to you for your quick fix :-)
